tests/test_model.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from shortlister import model
+from pathlib import Path
+
+path= Path("test_role")
+pickle_file_name = Path("shortlist.pickle")
+csv_file=Path("criteria.csv")
+
+def test_load_criteria():
+    criteria_result = model.load_criteria(path/csv_file)
+    expected = model.Criterion
+    for criterion in criteria_result:
+        assert type(criterion) is expected
+        assert criterion.name and criterion.description and criterion.scores is not None
+
+@pytest.mark.parametrize("folder_path,expected",
+                         [(path,["Emma Jones","Michael Davis","Sarah Thompson"]),
+                          ("non_existing_folder",[])])
+def test_load_applicants(folder_path,expected):
+    applicants = model.load_applicants(folder_path)
+    result = [applicant.name for applicant in applicants]
+    assert result == expected
+
+def test_load_role():
+    criteria = [model.Criterion(name="PhD",
```

Comment:
In tests, it's good idea to use terse dummy values so you are not distracted by real data. It is a lot to read, unnecessarily. For example, in your Criteria objects use instead:

```
scores = ("1", "2", "3")
criteria = [
    model.Criterion(name="c1", description="d1", scores=scores,
    model.Criterion(name="c2", description="d2", scores=scores,
    model.Criterion(name="c3", description="d3", scores=scores
]
```